src/Components/Shared/Header/Header.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+export default class header extends Component {
```

Comment:
I think you should call this Component a NavBar since that the function of it. It’s not a Header.